File: LogicadeProgramacao/Estrutura de Decisão/2. Java/FolhaPagamento.java
import java.util.Scanner;

public class FolhaPagamento {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Digite o valor da hora trabalhada: ");
        double valorHora = input.nextDouble();
        System.out.print("Digite a quantidade de horas trabalhadas no mês: ");
        double horasTrabalhadas = input.nextDouble();
        double salarioBruto = valorHora * horasTrabalhadas;
        double sindicato = salarioBruto * 0.03;
        double fgts = salarioBruto * 0.11;
        double ir = 0.0;
        double inss = salarioBruto * 0.10;
        double totalDescontos = 0.0;
        double salarioLiquido = 0.0;

        if (salarioBruto <= 900) {
            ir = 0.0;
        } else if (salarioBruto <= 1500) {
            ir = salarioBruto * 0.05;
        } else if (salarioBruto <= 2500) {
            ir = salarioBruto * 0.10;
        } else {
            ir = salarioBruto * 0.20;
        }

        totalDescontos = sindicato + ir + inss;
        salarioLiquido = salarioBruto - totalDescontos;

        System.out.printf("Salário Bruto: R$ %.2f%n", salarioBruto);
        System.out.printf("(-) IR: R$ %.2f%n", ir);
        System.out.printf("(-) INSS: R$ %.2f%n", inss);
        System.out.printf("(-) Sindicato: R$ %.2f%n", sindicato);
        System.out.printf("FGTS: R$ %.2f%n", fgts);
        System.out.printf("Total de descontos: R$ %.2f%n", totalDescontos);
        System.out.printf("Salário líquido: R$ %.2f%n", salarioLiquido);
    }
}

File: LogicadeProgramacao/Estrutura de Decisão/3. Python/FolhaPagamento.py
valor_hora = float(input("Digite o valor da sua hora de trabalho: "))
qtd_horas = float(input("Digite a quantidade de horas trabalhadas no mês: "))

salario_bruto = valor_hora * qtd_horas
fgts = salario_bruto * 0.11
desconto_inss = salario_bruto * 0.1

if salario_bruto <= 900:
    desconto_ir = 0
elif salario_bruto <= 1500:
    desconto_ir = salario_bruto * 0.05
elif salario_bruto <= 2500:
    desconto_ir = salario_bruto * 0.1
else:
    desconto_ir = salario_bruto * 0.2

total_descontos = desconto_ir + desconto_inss + (salario_bruto * 0.03)
salario_liquido = salario_bruto - total_descontos

print(f"Salário Bruto: R$ {salario_bruto:.2f}")
print(
    f"(-) IR ({desconto_ir / salario_bruto * 100:.0f}%): R$ {desconto_ir:.2f}")
print(f"(-) INSS (10%): R$ {desconto_inss:.2f}")
print(f"FGTS (11%): R$ {fgts:.2f}")
print(f"Total de descontos: R$ {total_descontos:.2f}")
print(f"Salário Líquido: R$ {salario_liquido:.2f}")
